modules/afamqp/afamqp.c

```diff
@@ -342,8 +384,41 @@ afamqp_dd_connect(AMQPDestDriver *self, gboolean reconnect)
       msg_error("Error allocating AMQP connection.");
       goto exception_amqp_dd_connect_failed_init;
     }
+  if(strcmp(self->ca_file,"") != 0)
```

Comment:
Thank you for your review.
I agree with you about implementing another way to switch off/on ssl instead of using ca_file, I used this other approach because i didn't want to excessively alter the code. 
Let's wait documentation team then.